yb-voyager/cmd/exportDataStatusCommand.go

```diff
@@ -38,16 +47,42 @@ type exportTableMigStatusOutputRow struct {
 
 // Note that the `export data status` is running in a separate process. It won't have access to the in-memory state
 // held in the main `export data` process.
-func runExportDataStatusCmd() error {
-	exportSchemaDoneFlagFilePath := filepath.Join(exportDir, "metainfo/flags/exportSchemaDone")
-	_, err := os.Stat(exportSchemaDoneFlagFilePath)
+func runExportDataStatusCmdDbzm() error {
+	err := checkSchemaExportDone()
 	if err != nil {
-		if errors.Is(err, os.ErrNotExist) {
-			return fmt.Errorf("cannot run `export data status` before schema export is done")
+		return err
+	}
+	exportStatusFilePath := filepath.Join(exportDir, "data", "export_status.json");
+	status, err := dbzm.ReadExportStatus(exportStatusFilePath)
+	if err != nil {
+		utils.ErrExit("Failed to read export status file %s: %v", exportStatusFilePath, err)
+	}
+	tables := status.Tables
+	sort.Slice(tables, func(i, j int) bool {
+		return tables[i].Sno < tables[j].Sno
+	})
+	var exportStatusOutputRows []*exportTableMigStatusOutputRow
+	for idx, table := range tables {
+		var status string
```

Comment:
nit: create var of type exportTableMigStatusOutputRow and use that only instead of extra var